spec/support/default_as.rb

```diff
@@ -0,0 +1,19 @@
+module DefaultAs
+  extend ActiveSupport::Concern
+
+  included do
+    let(:default_as) { :json }
+    prepend RequestHelpersCustomized
+  end
+
+  module RequestHelpersCustomized
+    l = lambda do |path, **kwargs|
+      kwargs[:as] ||= default_as if default_as
+      super(path, kwargs)
+    end
```

Comment:
Why separate the lambda from the define_method?  i.e. why not do...

```ruby
    %w[post patch put].each do |method|
      define_method(method) do |path, **kwargs|
        kwargs[:as] ||= default_as if default_as
        super(path, kwargs)
      end
    end
```

Reply:
Good question. I adapted this code from a snippet I found on SO, and after it worked I didn't want to mess with it 😛 . I'll change it.